src/main/java/by/itechart/film2nightApp/controller/AuthenticationController.java

```diff
@@ -0,0 +1,66 @@
+package by.itechart.film2nightApp.controller;
+
+import by.itechart.film2nightApp.dto.AuthenticationRequestDTO;
+import by.itechart.film2nightApp.entity.Role;
+import by.itechart.film2nightApp.entity.User;
+import by.itechart.film2nightApp.security.jwt.JWTTokenProvider;
+import by.itechart.film2nightApp.service.UserService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+@RestController
+@RequestMapping(value = "/api")
+public class AuthenticationController {
+
+
+    private final AuthenticationManager authenticationManager;
+    private final JWTTokenProvider jwtTokenProvider;
+    private final UserService userService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationManager authenticationManager, JWTTokenProvider jwtTokenProvider, UserService userService) {
+        this.authenticationManager = authenticationManager;
+        this.jwtTokenProvider = jwtTokenProvider;
+        this.userService = userService;
+    }
+
+   @PostMapping("/login")
+    public ResponseEntity login (@RequestBody AuthenticationRequestDTO authenticationRequestDTO){
+       try{
```

Comment:
too much logic for controller. You can move logic to service layer. And please insert spaces, e.g. between "try" and "{", "}" and "catch" and so on.